packages/cozy-sharing/src/components/ShareByEmail.jsx

```diff
@@ -120,22 +136,24 @@ export class ShareByEmail extends Component {
     }
   }
 
-  getSharingTypes = () => {
-    const { t } = this.props
-    const { sharingType } = this.state
-    const twoWay = {
-      value: 'two-way',
+  getSharingOptions = () => {
+    const { t, sharing, document } = this.props
+    const isSharingReadOnly = sharing
+      ? isReadOnlySharing(sharing, document._id)
+      : false
```

Comment:
For my culture, in which case can the `sharing` prop be undefined ?

Reply:
If the sharing is not created. We display the same modal for sharing creation and sharing edition